conf/modules/airspeed_sdp3x.xml

```diff
@@ -8,15 +8,18 @@
     <define name="SDP3X_I2C_DEV" value="i2cX" description="set i2c peripheral (default: i2c2)"/>
     <define name="SDP3X_I2C_ADDR" value="0x50" description="i2c slave address (default: 0x50)"/>
     <define name="SDP3X_SYNC_SEND" value="TRUE|FALSE" description="flag to enable sending every new measurement via telemetry (default:FALSE)"/>
-    <define name="SDP3X_PRESSURE_SCALE" value="1.0" description="pressure scaling factor to convert raw output to Pa (default: set according to pressure range according to datasheet)"/>
-    <define name="SDP3X_PRESSURE_OFFSET" value="0." description="pressure offset in Pa (default: set according to pressure range  according to datasheet)"/>
+    <define name="SDP3X_PRESSURE_SCALE" value="1.05" description="pressure scaling factor to convert raw output to Pa (default: set according to pressure range and output type according to datasheet)"/>
+    <define name="SDP3X_PRESSURE_OFFSET" value="8618.4" description="pressure offset in Pa (default: set according to pressure range and output type according to datasheet)"/>
```

Comment:
this example value is way to far from realistic values, better keep it to zero, which is the current default

Reply:
Corrected them to one and zero.